lib/app/modules/hotel_details/views/components/hotel_details_comments.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:rihlatic/app/core/components/images/network_image_component.dart';
+import 'package:rihlatic/app/core/styles/main_colors.dart';
+import 'package:rihlatic/app/core/styles/text_styles.dart';
+
+class HotelDetailsComments extends StatelessWidget {
+  const HotelDetailsComments({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Row(
+              children: [
+                ClipRRect(
+                  borderRadius: BorderRadius.circular(1000.r),
+                  child: SizedBox(
+                    width: 39.5.r,
+                    height: 39.5.r,
+                    child: NetworkImageComponent(
+                        imageLink:
+                            "https://images.pexels.com/photos/8993561/pexels-photo-8993561.jpeg?auto=compress&cs=tinysrgb&w=800"),
+                  ),
+                ),
+                SizedBox(
+                  width: 8.w,
+                ),
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      'Col. Roderick Decker',
+                      style: TextStyles.smallLabelTextStyle(context).copyWith(
+                        color: MainColors.textColor(context)!,
+                      ),
+                      maxLines: 2,
+                      textAlign: TextAlign.left,
+                    ),
+                    Text(
+                      '22 April, 2024',
+                      style: TextStyles.smallBodyTextStyle(context).copyWith(
+                        color: MainColors.textColor(context)!.withOpacity(0.6),
+                      ),
+                      textAlign: TextAlign.left,
+                    ),
+                  ],
+                ),
+              ],
+            ),
+            SizedBox(
+              height: 10.h,
+            ),
+            Text(
+              'Very convenient location just across the road from Paris Nord station...',
+              style: TextStyles.smallBodyTextStyle(context).copyWith(
+                  color: MainColors.textColor(context)!, fontSize: 15.r),
+            ),
+            SizedBox(
+              height: 10.h,
+            ),
+            Row(
```

Comment:
error in UI when there are more then 4 pictures, so try to fix this problem with listView or wrap or (show + in last picture)